algopy_testing/pyproject.toml

```diff
@@ -0,0 +1,33 @@
+[tool.poetry]
+name = "algorand-python-testing"
+version = "0.0.1"
+description = "Algorand Python Testing library"
+authors = ["Algorand Foundation <contact@algorand.foundation>"]
+readme = "README.md"
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Intended Audience :: Developers",
+    "Topic :: Software Development :: Testing",
+]
+packages = [
+    { include = "algopy", from = "src" }
+]
+
+[tool.poetry.dependencies]
+python = "^3.12"
+
+[tool.poetry.group.testing.dependencies]
+pytest = "^7.4.0"
+pytest-xdist = {extras = ["psutil"], version = "^3.3.1"}
+py-algorand-sdk = "^2.4.0"
+algokit-utils = "^2.2.1"
+pytest-cov = "^4.1.0"
+prettytable = "^3.9.0"
```

Comment:
at this stage I don't think we will need prettytable, it was used for one very specific thing in the puya tests